src/frontend/src/utils/urls.ts

```diff
@@ -22,8 +22,6 @@ const userSecureSettingsSet = () => `${users()}/secure-settings/set`;
 const userRoleByUserId = (id: string) => `${usersById(id)}/change-role`;
 const userFavoriteProjects = (id: string) => `${usersById(id)}/favorite-projects`;
 const userSecureSettings = (id: string) => `${usersById(id)}/secure-settings`;
-const userScheduleSettings = (id: string) => `${usersById(id)}/schedule-settings`;
-const userScheduleSettingsSet = () => `${users()}/schedule-settings/set`;
```

Comment:
Your builds are failing because you deleted a lot of the urls that need to be in the code